README.md

```diff
@@ -465,8 +465,90 @@ BsDBWriteAddress = ''
 Workers = 0
 
 [APIRateLimiter]
-# optional
-PathPattern = []
+# every line should represent one entry of gateway route. The comment after each line must contain which route name it represents.
+# Most of APIs has a qps number, offered by QA team.  That usually means the max qps for the whole 4 gateway cluster.
+# How to setup the RateLimit value, it is a sophistcated question and need take a lot of factors into account.
+# 1. For most query-APIs, we can setup a rate limit up to the 1/4 of max qps, as the config is for only one gateway instance.
+# 2. Also we avoid to setup a too large or too small rate limit value.
+# 3. For upload/download APIs, it is diffiult to use a rate limit as a protect mechanism for the servers. Because the performance of upload/download interactions usually dependens on how large the file is processed.
+# 4. We tetatively setup 50~75 as the rate limit for the download/upload APIs and we can ajdust them once we have a better experience.
+# 5. The rate limt config will upgraded in next version to use http methods and virtual-host/path style as part of the matching keys.
+
+# optional
```

Comment:
can you also add sample config for virtual host pattern

Reply:
Virtual host pattern does not need to be handled seperately, please refer to the new code in pkg/middleware/http/ratelimiter.go to see how to identify virtual host and modify virtual host pattern to match same pattern as path pattern.